src/main/resources/application.properties

```diff
@@ -1,28 +1,47 @@
 server.port=8809
+
 spring.datasource.url=jdbc:postgresql://${PGHOST}:${PGPORT}/${PG_DATABASE}
 spring.datasource.username=${PG_RW_USER}
 spring.datasource.password=${PG_RW_PASSWORD}
 spring.datasource.platform=staging
 spring.jpa.properties.hibernate.default_schema=staging
 spring.jpa.show-sql=true
 
+
+##spring.datasource.url=jdbc:postgresql://localhost:5431/sync_db
```

Comment:
we should not have these comments in here for local config, especially if we figure out local properties configuration